revolv/project/models.py

```diff
@@ -396,6 +397,15 @@ def amount_repaid(self):
         return self.adminrepayment_set.aggregate(models.Sum('amount'))["amount__sum"] or 0.0
 
     @property
+    def total_amount_to_be_repaid(self):
```

Comment:
Oh nice, I didn't even think that this would be different that the funding goal. Nice catch!